Procfile

```diff
@@ -1 +0,0 @@
-web: python -m girder --port ${PORT}
```

Comment:
Ok, I've filed https://github.com/girder/girder/issues/2078 , in case anyone wants to ever come back to it.